### ISSUE TITLE
eosio-abi - bool defined as _Bool

### ISSUE THREAD

**Issue by smlu** [EOSIO/eosio.cdt#99]
When generating abi via  `eosio-abi` boolean type is defined as `_Bool` and not as `bool`.

example:
```
[[eosio::action]]
void testaction(bool b1, bool b2)
{}
```

outputed abi:

```
....
{
    "name": "testaction",
        "base": "",
        "fields": [
            {
                "name": "b1",
                "type": "_Bool"
            },
            {
                "name": "b2",
                "type": "_Bool"
            }
        ]
}
```

**Comment by dskvr** [comment]
@larryk85 replicated this issue. This bug also arises when utilizing a boolean inside of a table's struct.

**Comment by Lucklyric** [comment]
It seems @larryk85 is refactoring the code of abigenerator. 
For the meanwhile, I temporarily solve the problem by adding:
`{"_Bool", "bool"}` mapping in` std::string translate_type( const clang::QualType& t ) `,
the file is located at `eosio.cdt/eosio_llvm/tools/clang/tools/extra/eosio_abigen_tool`.
After that, you can rebuild and reinstall it.

**Comment by larryk85** [comment]
fixed as of v1.3.0
